### ISSUE TITLE
No access to anything InfoButtons related!

### ISSUE THREAD

**Issue by DSSUSER** [logicahealth/InfoButtons#9]
InfoButtons was working on Friday.

Something changed over the weekend, or this morning.

We can no longer access ANY data, from ANY content provider.

No matter what drug we attempt to query, we get the following error:

"org.openinfobutton.exception.OIBProfileProcessingException: Access Check Error Caused By Configuration Problem In: MedlinePlus"

I tried to disable MedlinePlus inside of the LITE GUI interface, but we still get the same error, for all drugs.

We get nothing back from any content provider, just the above error.

Nothing has changed inside of our application.

No files were modified locally.

Joe Fisher
DSS Inc.
561-284-7631 (Office)

**Comment by aniskand** [comment]
That's very strange, especially if the profile itself was not modified.
It's possible there was some kind of database connection error or the
application itself through crashed due to running of memory. I would try
restarting the OIB server, while leaving MedLine plus disabled.

**Comment by gdelfiol** [comment]
Joe,

Are you still having problems? Our instance is working fine. This might have been a temporary issue with the MedlinePlus API, but it should have resolved after you disabled it.

**Comment by DSSUSER** [comment]
Thank you for the response.

I restarted Tomcat, and this error went away.

After restarting Tomcat, I can now access all of the content providers that are currently “enabled”.

However, there is still an issue, that for some reason did not resolve itself, after the Tomcat restart.

For those content providers that are currently enabled, when we click on the edit “pencil” icon, we get the following popup, with the checkbox.

[cid:image001.png@01D21274.E11064E0]

For those content providers that are NOT currently enabled, we do not see a checkbox, when we click on the edit “pencil” icon, so there is no way to re-enable the content provider:

[cid:image002.png@01D21275.BAACB500]

Note that because of the previous issue noted with the MedLine plus error, I disabled the MedLine Plus link, by “un-checking” the checkbox.

After bouncing Tomcat, I no longer get the Medline Plus error, but I do not get a checkbox, in any of the currently disabled content provider links.

What should I check next?

My documentation does not show any other services to restart.

Short of bouncing the entire server, is there anything else I need to check, or any other services that might need to be restarted?

Thanks in advance, and have a great afternoon.

Joseph C. Fisher | Linux Systems Engineer
Office: (561) 284-7631 | Cell: (561) 596-8566 | Fax: (813) 980-1637
jfisher@dssinc.commailto:jfisher@dssinc.com | vxsupport@dssinc.commailto:vxsupport@dssinc.com

[DSS_Logo_EHR_2013_800]http://www.dssinc.com/
5004 E. Fowler Ave Suite E
Tampa, FL 33617
